js/main.js

```diff
@@ -0,0 +1,18 @@
+const getRandomNumber = (MinimumValue, MaximumValue) => {
```

Comment:
```suggestion
const getRandomNumber = (MinimumValue, MaximumValue) => {
```
переменные просто так с большой буквы быть не должны

Reply:
Да это косяк, пользуюсь яндекс переводчиком и по привычке пишу с большой буквы новое слово.